aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -1486,6 +1487,26 @@ impl AptosVM {
         Ok(())
     }
 
+    /// Check whether the bytecode can be published to mainnet based on the unstable tag in the metadata
+    fn validate_unstable_bytecode(&self, modules: &[CompiledModule]) -> VMResult<()> {
```

Comment:
nit: `validate_compiled_metadata`? Otherwise the function name reads a bit weird... or you can call it `validate_no_unstable_bytecode`

Reply:
done